### PR TITLE
Feature/8/lom keywords in header

### PR DESCRIPTION
**Pull request by chfsx**
Hi @klees and @smeyer-ilias 

This PR contains the Feature ["Expose LOM keywords in HTML header"](https://docu.ilias.de/goto.php?target=wiki_1357_Expose_LOM_keywords_in_HTML_header), which brings changes for 

- UI Page
- Metadata
- GlobalScreen

Background of the Feature is the missing rendering of LOM Keywords in the HTML header since ILIAS 6.

- @klees: All changes for UI Page incl. new Tests are here: https://github.com/ILIAS-eLearning/ILIAS/commit/c48162e429cc3d884f1f82f50bd4661748b0237a
- @smeyer-ilias: All Changes for MetaData are here: https://github.com/ILIAS-eLearning/ILIAS/commit/2923d8918b8ec8e15ec20f4096b3f8f562f4a8be and https://github.com/ILIAS-eLearning/ILIAS/pull/4048/commits/40ae8136e1593a5068546f93d7e86991de2bedea

### CODE REVIEW

**Comment by chfsx** [comment]
@smeyer-ilias there will follow a part in the `ilMDKeywordExposer` for "language" and "copyright" as well. But currently I'm unable to access the metadata-administration in trunk to setup copyrights

**Review by klees — APPROVED** [review]
LGTM, please set this on the JF agenda for monday.

We might want to take a closer look into this header section in the UI framework at some point, but this is not it.

**Comment by chfsx** [comment]
@smeyer-ilias I found a workaround and implemented Language and Copyright: https://github.com/ILIAS-eLearning/ILIAS/pull/4048/commits/40ae8136e1593a5068546f93d7e86991de2bedea

**Comment by matthiaskunkel** [comment]
Jour Fixe, 21 FEB 2022: We highly appreciate this suggestion and accept the PR for ILIAS 8. Please add some information about this new options within the global screen service to the Readme file.

**Comment by chfsx** [comment]
@smeyer-ilias your feedback to the Metadata-party is highly welcome

**Comment by chfsx** [comment]
@smeyer-ilias thanks for the comments, I implemented this as desired here: https://github.com/ILIAS-eLearning/ILIAS/pull/4048/files#diff-7a2e645e59c8c7b6e41138323a5b62025f4749264b86bbfd93dfab743d89d4b8R33

An additional docu in the readme is here as well: https://github.com/ILIAS-eLearning/ILIAS/pull/4048/files#diff-7b9c08aac24ecfa955433a603f696336856d02cd78a4a21dba06da2efbd11ab2

**Comment by chfsx** [comment]
@smeyer-ilias if there are no other issues, you can approve the PR. I'll merge then after the customers approval

**Review by smeyer-ilias — APPROVED** [review]
Thanks Fabian, this looks good to me. 
Sorry, for the delay.

**Pull request merged**
